content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,61 +9,71 @@ weight: 20
+updating, and deleting primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE, GET). The API includes additional subresources for many objects that allow fine grained authorization (such as binding a pod to a node), and can accept and serve those resources in different representations for convenience or efficiency. Kubernetes also supports efficient change notifications on resources via "watches" and consistent lists, so that API clients can effectively cache, track, and synchronize the state of resources.
+
+## Kubernetes API terminology {#standard-api-terminology}
+
+Most Kubernetes API resource types are [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects): they represent a concrete instance of a concept on the cluster, such as a {{< glossary_tooltip term_id="pod" >}} or {{< glossary_tooltip term_id="namespace" >}}. A smaller number of API resource types are "virtual" - they often represent operations rather than objects, such as a permission check (use a POST with a JSON-encoded body of kind `SubjectAccessReview` to the `subjectaccessreviews` resource). All objects have a unique [names](/docs/concepts/overview/working-with-objects/names/#names) to allow idempotent creation and retrieval, but virtual resource types may not have unique names if they are not retrievable or do not rely on idempotency.
 
-## Standard API terminology
+Kubernetes relies on terminology from [ReST](https://en.wikipedia.org/wiki/Representational_state_transfer)
+to name and describe several API concepts:
 
-Most Kubernetes API resource types are [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects): they represent a concrete instance of a concept on the cluster, like a pod or namespace. A smaller number of API resource types are "virtual" - they often represent operations rather than objects, such as a permission check (use a POST with a JSON-encoded body of `SubjectAccessReview` to the `subjectaccessreviews` resource). All objects will have a unique name to allow idempotent creation and retrieval, but virtual resource types may not have unique names if they are not retrievable or do not rely on idempotency.
+* A **resource type** is the name used in the URL (for example: `pods`, `namespaces`, `services`);
+* All resource types have a concrete representation in JSON (their object schema) which is called a **kind**;
+* A set of instances of a resource type is known as a **collection**;
+* A single instance of the resource type is called an **object**, or sometimes a **resource**.
 
-Kubernetes generally leverages standard RESTful terminology to describe the API concepts:
+{{< note >}}
+In URLs, Kubernetes types such as `pod` and `configmap` are **lowercase**. This is different from
+how Kubernetes conventionally renders those types elsewhere, such as the `.kind` field of an
```

Comment:
The manifests is the exact location where the API specifications are codified. We should not call them "elsewhere". On the contrary, the CLI allows for a more ambiguous spelling for users convenience. Right?

Reply:
The comparison is between manifests and API URLs.

Reply:
If we really want to clarify the concepts, maybe one of the authoritative source of  this is this: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md

There you can find how the terms are defined:

> * Kind the name of a particular object schema (e.g. the "Cat" and "Dog" kinds would have different attributes and properties)
> * Resource a representation of a system entity, sent or retrieved as JSON via HTTP to the server. Resources are exposed via:
> * Collections - a list of resources of the same type, which may be queryable
> * Elements - an individual resource, addressable via a URL.

> Each resource typically accepts and returns data of a single kind. A kind may be accepted or returned by multiple resources that reflect specific use cases. For instance, the kind "Pod" is exposed as a "pods" resource that allows end users to create, update, and delete pods ...

Reply:
I've omitted this change from the latest revision.